airflow/ui/openapi-gen/requests/schemas.gen.ts

```diff
@@ -2823,11 +2823,18 @@ export const $HealthInfoSchema = {
       $ref: "#/components/schemas/TriggererInfoSchema",
     },
     dag_processor: {
-      $ref: "#/components/schemas/DagProcessorInfoSchema",
+      anyOf: [
+        {
+          $ref: "#/components/schemas/DagProcessorInfoSchema",
+        },
+        {
+          type: "null",
+        },
+      ],
```

Comment:
This is not accurate. `dag_processor` is not either `DagProcessorInfoSchema` or `None`. 

It's either defined and here with `DagProcessorInfoSchema` or completely missing from the body.

Reply:
To achieve that, you can take a look at fast api `response_model=Item, response_model_exclude_unset=True`

This should allow you to actually just return the plain dictionnary in your endpoint. (remove the returned type annotation as well of the endpoint)